working-groups/query-standardization/dsl-survey/ppl/README.md

```diff
@@ -0,0 +1,256 @@
+        * [MIN](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#min)
+        * [VAR_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-samp)
+        * [VAR_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#var-pop)
+        * [STDDEV_SAMP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-samp)
+        * [STDDEV_POP](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#stddev-pop)
+        * [TAKE](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/stats.rst#take)
+* **Timestamp Alignment**:
+    * Typically aligned to the start of the window.
+* **Boundaries**:
+    *  Inclusive of the start, exclusive of the end.
+* **Aggregation Functions for Downsampling**:
+    * N/A - can use OpenSearch specific engine downsampling via [Index roolup](https://opensearch.org/docs/latest/im-plugin/index-rollups/index/)
+* **Determining Correct Aggregation Type if Fixed**:
+    * Not applicable as it's not fixed.
+* **Querying Over Pre-Computed Temporal Aggregates**:
+    * Both manual and automatic selection is supported, depending on the query and time range.
+
+**Interpolation**
+
+* **Interpolation Scheme**:
+    * Depends on the OpenSearch settings and query configuration (schema on read).
+* **User Selection of Interpolation Scheme**:
+    *  Not typically user-configurable at query time.
+* **Handling Missing Data**:
+    * Represented internally as null, NaN, etc., depending on the context.
+* **Handling Missing Data for Aggregations**:
+    * Specific behavior depends on the aggregation function and data type.
+* **Querying Across Different Retention Cutoffs**:
+    * Managed at the data level; PPL doesn’t inherently handle this scenario.
+
+**Identifiers**
+
+* **Time Series Identifier Components**:
+    * Metric name, namespace, tags, etc., depending on how the data is modeled (See [Simple Schema For Observability](https://github.com/opensearch-project/opensearch-catalog/tree/main/docs/schema/observability/metrics) ).
+* **Supported Character Sets**:
+    *  Typically UTF-8.
+* **Dimensional Time Series ID Components**:
+    * Metric names can be separate or a dimension; dimensions can have multiple values (Also See API access to Prometheus ).
+* **Dimension Names, Requirements, and Types**:
+    * Supported based on how they are defined in the data model.
+* **Regular Expressions**:
+    * Supported
+* **Extraction Groups**:
+    * Depending on the data and query structure, they may be supported.
+
+*Keep in mind that OpenSearch PPL provides a querying interface, and some of the behaviors related to data modeling, aggregation, interpolation, and handling missing data might be influenced more by how the data is indexed, stored, and managed within OpenSearch rather than the PPL itself.*
```

Comment:
Absolutely. We do need some examples of those various behaviors based on the data models as we'll be evaluating standards against those highly technical details.

Reply:
Ppl is a language spec in opensearch-project 

It's implements a data source in as opensearch , s3 and spark via catalyst